src/pivot-table/hooks/use-disclaimer.ts

```diff
@@ -0,0 +1,39 @@
+import type { stardust } from "@nebula.js/stardust";
+import type { LayoutService } from "../../types/types";
+import { DEFAULT_FONT_FAMILY, DISCLAIMER_HEIGHT } from "../constants";
+import { useBaseContext } from "../contexts/BaseProvider";
+
+export interface DisclaimerProps {
+  translator: stardust.Translator;
+  layoutService: LayoutService;
+}
```

Comment:
I think this belongs to component not hook, because it originates from component and the hook is the consumer of that prop types

Reply:
Agree